internal/app/config.go

```diff
@@ -39,6 +40,21 @@ type Auth0Config struct {
 	WebClientID  string
 }
 
+type AuthConfig struct {
+	Domain string `default:"http://localhost:8080"`
+	Key    string
+	Pkix   AuthPkixConfig
+}
+
+type AuthPkixConfig struct {
```

Comment:
DN is better

- DN = Distinguished name (X.509 DN)
- PKIX = Public Key Infrastructure working group

```suggestion
type AuthDNConfig struct {
```